website/src/pages/index.js

```diff
@@ -79,6 +79,13 @@ Members of the React Native team frequently speak at various conferences.
 <br/><br/>
 You can follow the latest news from the React Native team on Twitter
   `,
+  introVideo: `
+The <a href="https://opensource.facebook.com/">Meta Open Source team</a> has put together a short overview of React Native, 
+where they explained the project in beginner's terms. You can also find 
+other content about Meta Open Source projects on their <a href="https://www.youtube.com/channel/UCCQY962PmHabTjaHv2wJzfQ">YouTube Channel</a>.
```

Comment:
I think this is odd to add -- I don't think coming to the React Native homepage folks are generally looking for other Meta Open Source projects. Can we remove the text "You can also find other content... You can follow the latest Meta OSS news.. " and just keep the twitter follow button?

Reply:
As well as re-ordering so the React Native talk video is on top as mentioned earlier?